product_docs/docs/tde/15/secure_key/key_rotation.mdx

```diff
@@ -4,29 +4,33 @@ description: Learn how to rotate your data encryption key.
 deepToc: true
 ---
 
-To change the master key, manually run the unwrap command specifying the old key. Then feed the result into the wrap command specifying the new key. Equivalently, if the data key is protected by a passphrase, to change the passphrase, run the unwrap command using the old passphrase. Then feed the result into the wrap command using the new passphrase. You can perform these operations while the database server is running. The wrapped data key in the file is used only on startup. It isn't used while the server is running.
+To change the master key, manually run the unwrap command, specifying the old key. Then feed the result into the wrap command, specifying the new key. 
+
+Equivalently, if the data key is protected by a passphrase, to change the passphrase, run the unwrap command using the old passphrase. Then feed the result into the wrap command using the new passphrase. 
```

Comment:
Could "Equivalently" be changed to "Alternatively"? If not, consider other ways to revise to simplify the language. Like

To achieve the same result, 

LMK which approach you like best.

Reply:
Actually, reading it again now, I think it doesn't need the adverb at all.